PerformanceMetrics/PerformanceMetrics.h

```diff
@@ -116,7 +124,7 @@ namespace Plugin {
         public:
             static constexpr char startURL[] = _T("about:blank");
 
-            virtual void LoadFinished(const string& URL, const int32_t httpstatus, const bool success, const uint32_t totalsuccess, const uint32_t totalfailed) = 0;
+            virtual void LoadFinished(const string& URL, const int32_t httpstatus, const bool success, const uint32_t totalsuccess, const uint32_t totalfailed, bool coldLaunch=false) = 0;
```

Comment:
This is the interface that is used by the monitoring engine part to communicate with the different reporting engines (systlog, tracing and perhaps in the future there might be more). By changing the interface and not adapting the tracing implementation that is now broken.
But the whole idea is to have the engine part now knowing about the details of what needs to be reported. The whole concept of a cold start is rather special and not all reporting engines might use that concept, therefore up to know it was only available in the syslog reporting engine. If there is a reason to change the coldstart reporting of course feel free to do so (it was implemented exactly as in the example and verified by QA) but please do not do it in the monitoring engine, that should not know about  these details, so please do so in the SyslogOutput.cpp so that other reporting engines are not forced to know about this detail and we keep the interface clean. I think you could just do it there as well but if not all info you need is available there reach out and we can see if we need to change the interface.

Reply:
Moved the cold launch logic internal to Syslogoutput.cpp.